### ISSUE TITLE
Convergence Issues with Dependencies

### ISSUE THREAD

**Issue by panghy** [OpenHFT/Chronicle-Map#94]
Seems like the current version (3.10.0)'s dependencies does not converge:

```
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability the error(s) are [
Dependency convergence error for net.openhft:chronicle-core:1.7.3 paths to dependency are:
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-core:1.7.3
and
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-threads:1.7.2
      +-net.openhft:chronicle-core:1.7.2
and
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-algorithms:1.1.7
      +-net.openhft:chronicle-core:1.3.0
, 
Dependency convergence error for net.openhft:chronicle-bytes:1.7.2 paths to dependency are:
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-values:1.5.3
      +-net.openhft:chronicle-bytes:1.7.2
and
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-wire:1.7.6
      +-net.openhft:chronicle-bytes:1.7.6
and
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-bytes:1.7.6
and
+-XXXX
  +-net.openhft:chronicle-map:3.10.0
    +-net.openhft:chronicle-algorithms:1.1.7
      +-net.openhft:chronicle-bytes:1.2.3
```

**Comment by leventov** [comment]
If you can just turn off the convergence check for the Chronicle Map dependency, you should probably do that, because it is known that despite the deps doesn't converge, everything is working if the latest chronicle-core and chronicle-bytes are used (and what maven does by default).

 This issue might be addressed only in the future versions

**Comment by panghy** [comment]
Alright, we just dependency managed all those divergences. Just thought we'd let you guys know about this issue.

**Comment by hft-team-city** [comment]
Released in Chronicle-Map-3.16.1 (https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.16.1), BOM-1.16.175 (https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-1.16.175)

**Comment by hft-team-city** [comment]
Released in Chronicle-Map-3.16.1 (https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.16.1), BOM-1.16.201 (https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-1.16.201)
